imports/plugins/core/ui/client/components/currencyInput/currencyInput.js

```diff
@@ -0,0 +1,72 @@
+import { unformat } from "accounting-js";
+import { registerComponent } from "@reactioncommerce/reaction-components";
+import TextField from "../textfield/textfield";
```

Comment:
For the sake of consistency with the components API, this should probably be...

```js
import { getRawComponent } from "@reactioncommerce/reaction-components";

const TextField = getRawComponent("TextField");

class CurrencyInput extends TextField {
...
``` 

That way if someone replaces the default `TextField` component with `replaceComponent`, you will inherit from the correct source when extending the class.  Probably a bit of a corner case that isn't likely to happen, but it would at least leave the option open.

Reply:
See the section called "Extending Components" in the Components API docs.

https://docs.reactioncommerce.com/reaction-docs/master/components-api